### ISSUE TITLE
multiqc did not use all the samples I provided

### ISSUE THREAD

**Issue by anna-miu** [ewels/MultiQC#1433]
Hi,

I ran MultiQC on FastQC outputs with paired samples. I have `149` samples (`x2`=`498` FastQC files). But MultiQC only used 193 files. I got a warning message as below:

```python
 /py_packages/3.7/lib/python3.7/site-packages/multiqc/plots/bargraph.py:451: UserWarning: FixedFormatter should only be ued together with FixedLocator
  axes.set_xticklabels(['{:.0f}%'.format(x) for x in vals])
```

What is the problem? How can I fix it?

Thanks very much.

**Comment by ewels** [comment]
Hi @anna-miu,

Did you check the docs for the troubleshooting section called _"Not enough samples found"_ yet? You can find it here: https://multiqc.info/docs/#not-enough-samples-found

The `FixedFormatter` thing was fixed in MultiQC v1.10 so I guess you're using an older version. If you update to the latest release (`v1.10.1`) then that warning should go away.

I'm not entirely sure where the `498` FastQC files come from? I guess this is a typo and you mean `298`? 😅 

Phil

**Comment by ewels** [comment]
Closing due to lack of response.
